flags/conditions.py

```diff
@@ -21,18 +25,22 @@ def register(condition_name, fn=None):
     when checking the flag state. """
     global CONDITIONS
 
-    # Don't be a decorator, just register
     if fn is None:
         # Be a decorator
         def decorator(fn):
             register(condition_name, fn=fn)
             return fn
         return decorator
 
-    if condition_name not in CONDITIONS:
-        CONDITIONS[condition_name] = []
+    # Don't be a decorator, just register
+    if condition_name in CONDITIONS:
+        raise DuplicateCondition(
+            'Flag condition "{name}" already registered.'.format(
```

Comment:
Is this going to happen for listed tuples, or just for `dict` conditions? In the docs above you indicate that names might be duplicated on purpose, so you'd only want to throw this error when the regexp _and_ the value are all the same?

Reply:
There are two sides to conditions, one where you can register new conditions to be used by feature flags, and the other where you provide the list of conditions that a flag must meet to be enabled. The registered conditions that are available to be used for feature flags can't have duplicate names, but the list of conditions a flag must meet can have duplicate condition names.